src/Layers/xrRenderPC_R1/FStaticRender_Loader.cpp

```diff
@@ -26,7 +26,7 @@ void CRender::level_Load(IReader* fs)
     g_pGamePersistent->LoadTitle();
     {
         chunk = fs->open_chunk(fsL_SHADERS);
-        R_ASSERT2(chunk, "Level doesn't builded correctly.");
+        R_ASSERT2(chunk, "Level isn't built correctly.");
```

Comment:
Don't do unrelated changes.
1. Since this is unrelated to this PR (which is dedicated only to CDB cache), the grammar can be fixed in separate PR.
2. But I think that original GSC text is canonical, so let's not touch it.
```suggestion
        R_ASSERT2(chunk, "Level doesn't builded correctly.");
```